### PR TITLE
Fixed bug #7962: Remove system functions from isql SHOW FUNC output

### PR DESCRIPTION
**Pull request by TreeHunter9**

### CODE REVIEW

**Comment by AlexPeshkoff** [comment]
As soon as you've started with bringing logic (and harmony) here please compare:

SQL> show func "RDB$BLOB_UTIL.SEEK";
There is no user-defined function RDB$BLOB_UTIL.SEEK in this database
SQL> show sys func "RDB$BLOB_UTIL.SEEK";
RDB$BLOB_UTIL.IS_WRITABLE
RDB$BLOB_UTIL.NEW_BLOB
RDB$BLOB_UTIL.OPEN_BLOB
RDB$BLOB_UTIL.READ_DATA
RDB$BLOB_UTIL.SEEK
RDB$PROFILER.START_SESSION
RDB$TIME_ZONE_UTIL.DATABASE_VERSION
SQL>

with:

SQL> show table "SEC$USERS";
SEC$USER_NAME                   (RDB$USER) CHAR(63) CHARACTER SET UTF8 Nullable 
SEC$FIRST_NAME                  (SEC$NAME_PART) VARCHAR(32) CHARACTER SET UTF8 Nullable 
SEC$MIDDLE_NAME                 (SEC$NAME_PART) VARCHAR(32) CHARACTER SET UTF8 Nullable 
SEC$LAST_NAME                   (SEC$NAME_PART) VARCHAR(32) CHARACTER SET UTF8 Nullable 
SEC$ACTIVE                      (RDB$BOOLEAN) BOOLEAN Nullable 
SEC$ADMIN                       (RDB$BOOLEAN) BOOLEAN Nullable 
SEC$DESCRIPTION                 (RDB$DESCRIPTION) BLOB segment 80, subtype TEXT CHARACTER SET UTF8 Nullable 
SEC$PLUGIN                      (RDB$PLUGIN) CHAR(63) CHARACTER SET UTF8 Nullable 
SQL>

I.e. there is no need to specify system table when providing particular system table name. I think same should work for functions.

**Comment by TreeHunter9** [comment]
> I.e. there is no need to specify 'system' for table when providing particular system table name. I think same should work for functions.

Yea, that makes sense. I'll fix it.

**Comment by TreeHunter9** [comment]
Btw, `show func "RDB$BLOB_UTIL.SEEK"` will not work, because it will think that `"RDB$BLOB_UTIL.SEEK"` is a function name. I thought about solution for this, maybe better to use syntax like this - `"RDB$BLOB_UTIL"."SEEK"`, but still it's kinda hard to parse, because we can have ether `"` and `.` in package/function name.
Anyway, this is not related to this PR, just my thoughts.

**Comment by AlexPeshkoff** [comment]
> Btw, `show func "RDB$BLOB_UTIL.SEEK"` will not work, because it will think that `"RDB$BLOB_UTIL.SEEK"` is a function name. I thought about solution for this, maybe better to use syntax like this - `"RDB$BLOB_UTIL"."SEEK"`, but still it's kinda hard to parse, because we can have ether `"` and `.` in package/function name. Anyway, this is not related to this PR, just my thoughts.

Really unrelated. Btw, I do not see big problems with parsing it - double quote (") is already taken into an account (general rule is used - quoted text "QQ""ZZ" represents name QQ"ZZ), appears enough to add check for dot (.) after quoted text.
